test/JDBC/expected/sys-all_columns-vu-verify.out

```diff
@@ -34,3 +34,50 @@ intcol#!#1#!#4#!#10#!#0#!#<NULL>#!#1#!#0#!#0#!#0#!#0#!#0#!#0#!#0#!#0#!#0#!#0#!#0
 moneycol#!#5#!#8#!#19#!#4#!#<NULL>#!#1#!#0#!#0#!#0#!#0#!#0#!#0#!#0#!#0#!#0#!#0#!#0#!#0#!#0#!#0#!#0#!#0#!#NOT_APPLICABLE
 ~~END~~
 
+
+SELECT is_computed FROM sys.all_columns WHERE name = 'col_computed'
```

Comment:
Modify the test like this to avoid so many lines:
```
SELECT name, is_computed, is_identity
FROM sys.all_columns
WHERE name in ('col_computed', 'col_three', 'col_two', 'col_two')
ORDER BY name